schema/bom-1.6.proto

```diff
@@ -148,6 +148,10 @@ message Component {
   repeated OrganizationalContact authors = 29;
   // Textual strings that aid in discovery, search, and retrieval of the associated object. Tags often serve as a way to group or categorize similar or related objects by various attributes. Examples include "json-parser", "object-persistence", "text-to-image", "translation", and "object-detection".
   repeated string tags = 30;
+  // Specifies the OmniBOR Artifact ID. The OmniBOR, if specified, MUST be valid and conform to the specification defined at: https://www.iana.org/assignments/uri-schemes/prov/gitoid
+  optional string omnibor = 31;
```

Comment:
better call it `omniborid` or something.
Omnivor is a system, a collection of things and processes and such , and the `OmniBOR ID` is the actial identifier string - according to https://omnibor.io/glossary/omnibor/#omnibor-identifier

Reply:
Changed omnibor to omniborId